File: demo/src/main/java/me/shkschneider/skeleton/demo/data/ShkMod.java
package me.shkschneider.skeleton.demo.data;

import com.google.gson.JsonArray;
import com.google.gson.annotations.SerializedName;

public class ShkMod {

    @SerializedName("api")
    public Integer api;
    @SerializedName("buildId")
    public String buildId;
    @SerializedName("revision")
    public String revision;
    @SerializedName("securityPatch")
    public String securityPatch;
    @SerializedName("ro.mod.name")
    public String roModName;
    @SerializedName("ro.mod.verison")
    public String roModVersion;
    @SerializedName("devices")
    public JsonArray devices;

}

File: demo/src/main/java/me/shkschneider/skeleton/demo/data/ShkMod.kt
package me.shkschneider.skeleton.demo.data

import com.google.gson.annotations.SerializedName

data class ShkMod(
        @SerializedName("api")
        val api: Int? = null,
        @SerializedName("buildId")
        val buildId: String?,
        @SerializedName("revision")
        val revision: String?,
        @SerializedName("securityPatch")
        val securityPatch: String?,
        @SerializedName("ro.mod.name")
        val roModName: String?,
        @SerializedName("ro.mod.version")
        val roModVersion: String?,
        @SerializedName("devices")
        val devices: List<String>?
)
